src/views/setup/components/selectEspIdf.vue

```diff
@@ -137,6 +152,28 @@ export default class SelectEspIdf extends Vue {
   set showGithubTags(showTags: boolean) {
     this.setShowIdfTagList(showTags);
   }
+  get isVersionLowerThan5() {
+    if (this.selectedIdfVersion && this.selectedIdfVersion.name) {
+      // Regular expression to match the version number in the format vX.X.X or release/vX.X
+      const match = this.selectedIdfVersion.name.match(/v(\d+(\.\d+)?(\.\d+)?)/);
```

Comment:
Any reason why not using the version validation in the helper function in `.../store/index.ts` ?